src/gpuarray/buffer.h

```diff
@@ -689,6 +689,13 @@ GPUARRAY_PUBLIC gpucontext *gpukernel_context(gpukernel *k);
  */
 #define GA_CTX_PROP_PCIBUSID 19
 
+/**
+ * Get the largest single block of memory that can be allocted.
+ *
+ * Type: `size_t`
+ */
+#define GA_CTX_PROP_LARGEST_MEMBLOCK 20
```

Comment:
I can add it for sure, but I usually only add the values that I need, and I don't need this one currently.